web_src/js/features/repo-commit.js

```diff
@@ -1,72 +1,69 @@
-import $ from 'jquery';
 import {createTippy} from '../modules/tippy.js';
 import {toggleElem} from '../utils/dom.js';
-
-const {csrfToken} = window.config;
+import {parseDom} from '../utils.js';
+import {POST} from '../modules/fetch.js';
 
 export function initRepoEllipsisButton() {
-  $('.js-toggle-commit-body').on('click', function (e) {
-    e.preventDefault();
-    const expanded = $(this).attr('aria-expanded') === 'true';
-    toggleElem($(this).parent().find('.commit-body'));
-    $(this).attr('aria-expanded', String(!expanded));
-  });
+  for (const button of document.querySelectorAll('.js-toggle-commit-body')) {
+    button.addEventListener('click', function (e) {
+      e.preventDefault();
+      const expanded = this.getAttribute('aria-expanded') === 'true';
+      toggleElem(this.parentElement.querySelector('.commit-body'));
+      this.setAttribute('aria-expanded', String(!expanded));
+    });
+  }
 }
 
-export function initRepoCommitLastCommitLoader() {
+export async function initRepoCommitLastCommitLoader() {
   const entryMap = {};
 
-  const entries = $('table#repo-files-table tr.notready')
-    .map((_, v) => {
-      entryMap[$(v).attr('data-entryname')] = $(v);
-      return $(v).attr('data-entryname');
-    })
-    .get();
+  const entries = Array.from(document.querySelectorAll('table#repo-files-table tr.notready'), (v) => {
+    entryMap[v.getAttribute('data-entryname')] = v;
+    return v.getAttribute('data-entryname');
+  });
 
   if (entries.length === 0) {
     return;
   }
 
-  const lastCommitLoaderURL = $('table#repo-files-table').data('lastCommitLoaderUrl');
```

Comment:
Lines above this conflict because of https://github.com/go-gitea/gitea/pull/29272, likely you can just copy all lines up to this statement.

Reply:
I see you merged "yours". I find my `for` loop slightly more efficent as it calls `getAttribute` only once instead of twice. Care to reconsider?

Reply:
Yours was using jQuery, feel free to push here though

Reply:
https://github.com/go-gitea/gitea/pull/29230/files#r1499558287 should do it, it's rather trivial to eliminate one call with a variable.